pom.xml

```diff
@@ -103,7 +103,7 @@
         <zookeeper.version>3.4.6</zookeeper.version>
         <jersey.version>1.9</jersey.version>
         <!-- align with org.apache.tinkerpop:tinkerpop -->
-        <netty4.version>4.1.25.Final</netty4.version>
+        <netty4.version>4.1.36.Final</netty4.version>
```

Comment:
Netty was upgraded to `4.1.42` in TinkerPop `3.4.5`: https://github.com/apache/tinkerpop/blob/master/CHANGELOG.asciidoc#tinkerpop-345-release-date-february-3-2020

Also, we bumped Netty to `4.1.45` here: https://github.com/JanusGraph/janusgraph/pull/2003 so it will be downgrade as I understand

Reply:
Thanks for checking this out @porunov. I noticed `master` was on `3.4.6` but `v0.4` is still on TP `3.4.4`. I'm a bit out of date, do you know what the TP strategy is for aligning versions between JG and TP? Should `v0.4` also be bumped up to `3.4.6`?

Reply:
Ah. Sorry, I missed that this PR is for `v0.4`. This thing is that we don't support `v0.4` anymore, but I am OK to push some easy fixes there if the community needs it (but we won't make any new releases from `v0.4` anymore).

Reply:
> Should `v0.4` also be bumped up to `3.4.6`?

 no we shouldn't bump TP in `v0.4` because we don't support that branch anymore but if someone needs it, I am OK with bumping TP to a newer version in `v0.4`. The only thing is that we won't produce new versions from `v0.4` branch, so the person who needs that fix should compile janusgraph.